test/note_display.test.js

```diff
@@ -0,0 +1,530 @@
+/* eslint-disable no-unused-expressions */
+/* eslint-disable no-undef */
+import * as Factory from './lib/factory.js';
+chai.use(chaiAsPromised);
+const expect = chai.expect;
+
+describe('note display criteria', function () {
```

Comment:
`note_display_criteria.test.js` seems to more accurately reflect the contents of this file?